public/css/style.css

```diff
@@ -422,6 +422,30 @@ body {
   }
 }
 
+.container_row{
+  display: grid;
+}
+
+.logo-back {
+  grid-column: 1;
+  grid-row: 1;
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo-front {
+  grid-column: 1;
+  grid-row: 1;
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
```

Comment:
You would normally only need to use tailwind utilities in the html and build the css file with the npm script so it uses the utility classes but if it works I can leave it there for now